Sources/StreamChat/Query/ChannelListQuery.swift

```diff
@@ -118,7 +118,16 @@ public extension FilterKey where Scope: AnyChannelListFilterScope {
 
     /// Filter for checking whether current user has muted the channel
     /// Supported operators: `equal`
-    static var muted: FilterKey<Scope, Bool> { .init(rawValue: "muted", keyPathString: #keyPath(ChannelDTO.mute)) }
+    static var muted: FilterKey<Scope, Bool> { .init(
+        rawValue: "muted",
+        keyPathString: #keyPath(ChannelDTO.mute),
+        predicateMapper: { muted in
+            if muted {
+                return NSPredicate(format: "mute != nil")
+            }
+            return NSPredicate(format: "mute == nil")
+        }
+    ) }
```

Comment:
Hum 🤔 I think that the `PredicateMapper` might need to have as an input of the closure the operation type as well 🤔  Because right now, this one works fine, because the "muted" only works for equality, but we might have other cases where we need to change the predicate based on if the operation is `equal` , `greater` etc...